### PR TITLE
RT-1.7: Local BGP Test- OTG: Adding BGP-PEER_GROUP configs to local_bgp_test.go

### PR DESCRIPTION
**Pull request by stthummuri**
Adding BGP-PEER_GROUP configs since bgp neighborship would not come up without those changes

### CODE REVIEW

File: feature/bgp/tests/local_bgp_test/local_bgp_test.go [hunk 1]
```diff
@@ -288,7 +335,7 @@ func TestParameters(t *testing.T) {
 				Timers: &oc.NetworkInstance_Protocol_Bgp_Neighbor_Timers{
 					HoldTime:           ygot.Uint16(holdTime),
 					NegotiatedHoldTime: ygot.Uint16(135),
-					KeepaliveInterval:  ygot.Uint16(keepAlive),
+					KeepaliveInterval:  ygot.Uint16(45),
```
Comment (bstoll) [inline feature/bgp/tests/local_bgp_test/local_bgp_test.go:338]: Shouldn't dutConf.Timers.KeepaliveInterval be the same as wantState.Timers.KeepaliveInterval ?  I don't think it is correct to just change this one value without also changing the other.

File: feature/bgp/tests/local_bgp_test/local_bgp_test.go [hunk 2]
```diff
@@ -59,9 +59,22 @@ func bgpWithNbr(as uint32, routerID string, nbr *oc.NetworkInstance_Protocol_Bgp
...
+	pg := bgp.GetOrCreatePeerGroup(peerGrpName)
+	pg.PeerAs = ygot.Uint32(*nbr.PeerAs)
+	pg.PeerGroupName = ygot.String(peerGrpName)
 	bgp.AppendNeighbor(nbr)
 	return bgp
 }
+func configureNIType(t *testing.T) {
+	dut := ondatra.DUT(t, "dut1")
+	ate := ondatra.DUT(t, "dut2")
+	// Configure Network instance type on DUT
```
Comment (greg-dennis) [inline feature/bgp/tests/local_bgp_test/local_bgp_test.go:73]: nit: period end of comment sentence

File: feature/bgp/tests/local_bgp_test/local_bgp_test.go [hunk 3]
```diff
@@ -304,7 +358,7 @@ func TestParameters(t *testing.T) {
 			// Renable and wait to establish
 			gnmi.Replace(t, dut, dutConfPath.Config(), tc.dutConf)
 			gnmi.Replace(t, ate, ateConfPath.Config(), tc.ateConf)
-			gnmi.Await(t, dut, nbrPath.SessionState().State(), time.Second*30, oc.Bgp_Neighbor_SessionState_ESTABLISHED)
+			gnmi.Await(t, dut, nbrPath.SessionState().State(), time.Second*60, oc.Bgp_Neighbor_SessionState_ESTABLISHED)
```
Comment (greg-dennis) [inline feature/bgp/tests/local_bgp_test/local_bgp_test.go:361]: if you want, time.Seconds*60 could just be time.Minute

**Comment by greg-dennis** [comment]
@stthummuri, ping to address above comments
@bstoll, @shidzhan pinging for review

**Comment by stthummuri** [comment]
Closing pull request since the changes got merged as part of https://github.com/openconfig/featureprofiles/pull/730/

**Pull request closed without merging**